src/runtime/api.js

```diff
@@ -22,15 +22,33 @@ module.exports = function(useSourceMap) {
 
   // import a list of modules into the list
   // eslint-disable-next-line func-names
-  list.i = function(modules, mediaQuery) {
+  list.i = function(modules, mediaQuery, dedupe) {
```

Comment:
Is that possible to have it to be `true` by default?
e.g.
`function(modules, mediaQuery, dedupe = true)`?

Reply:
Why?